scripts/helper/create-topics.sh

```diff
@@ -1,47 +1,18 @@
 #!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
 
-# Create Kafka topic users, using appSA principal
-export KAFKA_LOG4J_OPTS="-Dlog4j.rootLogger=DEBUG,stdout -Dlog4j.logger.kafka=DEBUG,stdout" && kafka-topics \
-   --bootstrap-server kafka1:11091 \
-   --command-config /etc/kafka/secrets/appSA.config \
-   --topic users \
-   --create \
-   --replication-factor 2 \
-   --partitions 2 \
-   --config confluent.value.schema.validation=true
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+cd $DIR
 
-# Create Kafka topics with prefix wikipedia, using connectorSA principal
-export KAFKA_LOG4J_OPTS="-Dlog4j.rootLogger=DEBUG,stdout -Dlog4j.logger.kafka=DEBUG,stdout" && kafka-topics \
-   --bootstrap-server kafka1:11091 \
-   --command-config /etc/kafka/secrets/connectorSA_without_interceptors_ssl.config \
-   --topic wikipedia.parsed \
-   --create \
-   --replication-factor 2 \
-   --partitions 2 \
-   --config confluent.value.schema.validation=true
-export KAFKA_LOG4J_OPTS="-Dlog4j.rootLogger=DEBUG,stdout -Dlog4j.logger.kafka=DEBUG,stdout" && kafka-topics \
-   --bootstrap-server kafka1:11091 \
-   --command-config /etc/kafka/secrets/connectorSA_without_interceptors_ssl.config \
-   --topic wikipedia.parsed.count-by-domain \
-   --create \
-   --replication-factor 2 \
-   --partitions 2
-export KAFKA_LOG4J_OPTS="-Dlog4j.rootLogger=DEBUG,stdout -Dlog4j.logger.kafka=DEBUG,stdout" && kafka-topics \
-   --bootstrap-server kafka1:11091 \
-   --command-config /etc/kafka/secrets/connectorSA_without_interceptors_ssl.config \
-   --topic wikipedia.failed \
-   --create \
-   --replication-factor 2 \
-   --partitions 2
+topics=(appSA.config users \
+        connectorSA_without_interceptors_ssl.config wikipedia.parsed \
+        connectorSA_without_interceptors_ssl.config wikipedia.parsed.count-by-domain \
+        connectorSA_without_interceptors_ssl.config wikipedia.failed \
+        ksqlDBUser_without_interceptors_ssl.config WIKIPEDIABOT \
+        ksqlDBUser_without_interceptors_ssl.config WIKIPEDIANOBOT \
+        ksqlDBUser_without_interceptors_ssl.config EN_WIKIPEDIA_GT_1 \
+        ksqlDBUser_without_interceptors_ssl.config EN_WIKIPEDIA_GT_1_COUNTS \
+       )
 
-# Create Kafka topics with prefix WIKIPEDIA or EN_WIKIPEDIA, using ksqlDBUser principal
-for t in WIKIPEDIABOT WIKIPEDIANOBOT EN_WIKIPEDIA_GT_1 EN_WIKIPEDIA_GT_1_COUNTS
-do
-  export KAFKA_LOG4J_OPTS="-Dlog4j.rootLogger=DEBUG,stdout -Dlog4j.logger.kafka=DEBUG,stdout" && kafka-topics \
-     --bootstrap-server kafka1:11091 \
-     --command-config /etc/kafka/secrets/ksqlDBUser_without_interceptors_ssl.config \
-     --topic "$t" \
-     --create \
-     --replication-factor 2 \
-     --partitions 2
-done
+printf '%s\0' "${topics[@]}" | xargs -0 -n2 -P15 sh -c 'echo "Creating topic $2 with principal $1";./create-topic.sh "$1" "$2";echo "Created topic $2";' sh
```

Comment:
@javabrett if topic creation unexpectedly fails for whatever reason, the script doesn't detect or fail fast.  e.g. I hacked the script to reference a `command-config` file that didn't exist.  There was a `Exception in thread "main" java.nio.file.NoSuchFileException` printed to stdout, but the script kept going.  The script should detect these errors...and/or...afterwards validate all topics were created (`kafka-topics --list...`).